src/components/VmsList/Vms.js

```diff
@@ -47,7 +47,7 @@ class Vms extends React.Component {
       <InfiniteScroll
         loadMore={this.loadMore}
         hasMore={vms.get('notAllPagesLoaded')}
-        loader={<div className={style['loaderBox']}><div className={style['loader']} /></div>}
+        loader={<div key={0} className={style['loaderBox']}><div className={style['loader']} /></div>}
```

Comment:
Same.

Reply:
This key doesn't matter since the loader is simply shifted on and off the scroller's children array, but I set it to a string to be type happy.